frontend/src/components/filters/FiltersWarningTS.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { QuestionCircleOutlined } from "@ant-design/icons";
+import { Tooltip, Typography } from "antd";
+import FiltersInfo from "./FiltersInfoTS";
+const { Text } = Typography
+
+interface FiltersWarning {
+    filters: any,
+    nFilters: number
+}
+
+const FiltersWarning = ({ filters, nFilters }: FiltersWarning) => {
+    if (nFilters === 0)
```

Comment:
Let's implement the FiltersBar component (see below) and handle this nFilters stuff internally so the caller doesn't have to mess around counting the number of filters for this component.